src/Crypto/core-crypto.asd

```diff
@@ -55,5 +55,48 @@ THE SOFTWARE.
                  "s-base64"
                  "emotiq"
                  "emotiq/delivery"
+                 "cffi"
+		 "core-crypto/libraries"
+                 "crypto-pairings/libraries"
                  ))
 
+(defsystem "core-crypto/libraries"
```

Comment:
Probably better to start a new hierarchy for the various external libraries that `core-crypto` needs.

Perhaps:

 "core-crypto/libraries" --> "core-crypto/lib/curve1174"
"crypto-pairings/libraries" --> "crypto-pairings/lib/pbc"

Reply:
good idea